src/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Apple</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Quicksand:300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <nav>
+      <ul>
+        <li><img src="../imgs/apple-icon.png" alt="apple icon" style="width: 20px; height: 17px;" /></li>
+        <li>Store</li>
+        <li>Mac</li>
+        <li>iPod</li>
+        <li>iPhone</li>
+        <li>iPad</li>
+        <li>iTunes</li>
+        <li>Support</li>
+        <li><input type="search"></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <div class="title">
+      <h1>What will your verse be?</h1>
+      <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</h2>
+
+      <div class="h3-wrapper">
+      <h3>Watch the "Your Verse" film and explore the stories </h3><img src="../imgs/play-button.svg" alt="play icon" style="width: 25px; -webkit-filter: invert(100%); filter: invert(100%); " class="play-icon">
+      </div>
+    </div>
+  </main>
+  <aside>
+    <div class="aside-img">
+      <img src="../imgs/img-1.jpg" alt="first image" />
```

Comment:
# wrap ```img```  tag in link tag as suggested in my main review 🙂

Reply:
I was so focused on making this _look_ like the source that I had forgotten to turn it into a link. Thank you so much for pointing this out. I'll be sure to keep a more detailed look next time!